modules/AOP_Case_Updates/AOP_Case_Updates.php

```diff
@@ -246,8 +259,9 @@ private function populateTemplate(EmailTemplate $template, $addDelimiter = true,
         $body = aop_parse_template(str_replace('$sugarurl', $sugar_config['site_url'], $template->body_html), $beans);
         $bodyAlt = aop_parse_template(str_replace('$sugarurl', $sugar_config['site_url'], $template->body), $beans);
         if ($addDelimiter) {
-            $body = $app_strings['LBL_AOP_EMAIL_REPLY_DELIMITER'].$body;
-            $bodyAlt = $app_strings['LBL_AOP_EMAIL_REPLY_DELIMITER'].$bodyAlt;
+            if ( $txtDelimiter === "" ) $txtDelimiter = $app_strings['LBL_AOP_EMAIL_REPLY_DELIMITER']; 
+            $body = $txtDelimiter.$body;
+            $bodyAlt = $txtDelimiter.$bodyAlt;
```

Comment:
Please make sure your code is compliant with [PSR 2](https://github.com/php-fig/fig-standards/blob/master/accepted/PSR-2-coding-style-guide.md)

A lot of the older code isn't, but any newly added code should.

This includes always using braces and correct spacing.